README.md

```diff
@@ -109,6 +109,23 @@ const mobilitybox = new Mobilitybox('abc');
 var trip = await mobilitybox.get_trip({id: "vesputi:trip:foobar"})
 ```
 
+#### Mobilitybox.find_trip_by_characteristics({origins_from_station, destination_station, origins_from_departure_time, destination_arrival_time, [line_name]})    
+returns with a *MobilityboxTrip* object based on its meta information.
```

Comment:
Actually it returns a promise that in turn returns a MobilityboxTrip object. We should clarify that.